### DISCUSSION TITLE
Why start cmd window?

### DISCUSSION THREAD

**Discussion by BayJay** [devlinx9/muon-ssh#92] [Q&A] [answered]
When I run Muon, a cmd window starts that does not close. How to remove it? it gets in the way

**Comment by devlinx9** [comment]
This is resolved in the new version

**Reply by BayJay** [reply]
last version 2.2.0 on Feb 8, 2022. 
when will the new version be released?
Love your program)

**Reply by devlinx9** [reply, marked as answer]
Now is available the new version 2.3.0, please check it and let me know.

**Reply by BayJay** [reply]
Good! Thank you. All OK
